sys/freebsd/kqueue.txt.const

```diff
@@ -1,5 +1,5 @@
 # Code generated by syz-sysgen. DO NOT EDIT.
-arches = 386, amd64
+arches = 386, amd64, arm64
```

Comment:
Are all consts really the same between all arches?

Reply:
There might be a few special cases that are MD but for the most part constants (system call numbers, ioctls, etc.) will be identical across architectures. These kqueue constants for example are in sys/sys/event.h.